internal/godocfx/parse.go

```diff
@@ -310,6 +319,16 @@ func parse(glob string, workingDir string, optionalExtraFiles []string, filter [
 	}, nil
 }
 
+// getStatus returns a possibly empty status string for the given
+// docs.
+func getStatus(doc string) string {
+	deprecated := "\nDeprecated:"
```

Comment:
Would it always work with a newline prefix? Is it possible that it's embedded within a line and only has a whitespace prefix instead?

Reply:
We may even be able to strengthen this to be two newlines before, but I think this is fine. See https://github.com/golang/go/wiki/Deprecated.